content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: blog
+title: "Introducing Kueue"
+date: 
+slug:
+---
+
+**Authors:** Abdullah Gharaibeh (Google), Aldo Culquicondor (Google)
+
+Whether on-premises or in the cloud with autoscaling capabilities, the reality is that clusters have finite capacity:
+constraints on resource usage exist for quota and cost management and so users will want an easy way to fairly and 
+efficiently share resources. 
+
+In this article we introduce [Kueue](https://github.com/kubernetes-sigs/kueue/tree/main/docs#readme),
+an open source job queueing controller designed to manage batch jobs as a single unit,
+while leaving pod-level orchestration to existing stable components of Kubernetes.
+Kueue natively supports the Kubernetes [Job](/docs/concepts/workloads/controllers/job/)
+API and offers hooks for integrating other custom built APIs for batch jobs.
+
+## What problem does Kueue solve?
+Job queueing is a key need for running batch workloads at scale in both on-prem and cloud environments. The central aspect of job queueing is
+managing access to a limited pool of resources shared by multiple tenants by deciding which jobs should wait and which can start now and what 
+resources they can use.
+
+Some of the most desired job queueing requirements include:
+- Quota and budgeting to control who can use what and up to what limit. This is not only needed in clusters with static resources like on-prem, 
+  but it is also needed in cloud environments to control spend or usage of scarce resources.
+- Fair sharing of resources between tenants. To maximize the usage of available resources, any unused quota assigned to inactive tenants should be 
+  allowed to be shared fairly between active tenants.
+- Flexible placement of jobs across different resource types based on availability. This is important in cloud environments which have heterogeneous 
+  resources such as different architectures (GPU or CPU models) and different provisioning modes (spot vs on-demand).
+- Support for autoscaled environments where resources can be provisioned on demand.
+
+## Why a new controller? 
+Plain Kubernetes doesn’t address the above requirements. In normal circumstances, once a Job is created, the job-controller instantly creates the 
+pods and kube-scheduler continuously attempts to assign the pods to nodes. At scale, this situation can work the control plane to death. There is 
+also currently no good way to control at the job level which jobs should get which resources first, and no way to express order or fair sharing. The 
+current ResourceQuota model is not a good fit for our needs as quotas are enforced on resource creation, and so there is no queueing of requests. The 
```

Comment:
```suggestion
current ResourceQuota model is not a good fit for these needs because quotas are enforced on resource creation, and there is no queueing of requests. The 
```